### DISCUSSION TITLE
Make all warning flags in terminal the same color

### DISCUSSION THREAD

**Discussion by nifte** [tailwindlabs/tailwindcss#5364] [Ideas]
Make all of the warnings in the terminal the same orange color for aesthetics and consistency :)

![warn](https://user-images.githubusercontent.com/9247037/131889318-1b19d7f8-a5a0-4365-8c9f-3263a8a0d89d.png)

**Comment by nifte** [comment]
Submitted pull request #5412

**Comment by Thomasmark5** [comment]
Oh so sorry but i am not sure about that... Thanks !! [mypascoconnect](https://www.mypascoconnect.info/)

**Reply by nifte** [reply]
?

**Comment by NovusOpus** [comment]
Edit: Op is Right, my commend was inaccurate by claiming one of the warnings was generated by **NextJS**

**Reply by nifte** [reply]
Pretty sure its from Tailwind's [log utility](https://github.com/tailwindlabs/tailwindcss/blob/master/src/util/log.js)
